packages/as-proba/assembly/probability/sampler.ts

```diff
@@ -1,32 +1,46 @@
 import { randomInt } from './random';
 
 /**
- * Generates samples based on a probability distribution.
+ * This module exports a class named Sampler that allows the generation of observations based on a
+ * probability distribution.
  *
- * This class shall be extended with your own probability.
+ * @remarks
+ * - This class shall be extended with your own probability.
+ * - Two methods are available to generate observations: {@link rejectionSampling} and
+ * {@link inverseCumulativeDistributionSampling}.
+ *    These methods use different techniques to generate observations and require the implementation
+ *    of a probability function.
  */
 export class Sampler {
   _bounderies: Float64Array;
 
   /**
-   * Instanciates a sampler.
+   * Creates an instance of the Sampler class.
    *
-   * This constructor calls seedRandom.
+   * @remarks
+   * - The constructor uses an optional seed for the Math.random() function.
+   * - The _bounderies field is initialized with an empty Float64Array.
```

Comment:
@privateRemarks ? (since its a private field, developpers will not see this field)